docs: change README (#25)
kodingdotninja/chakra-icons: README.md

- Create Issue, Request Features, & send a pull request.

> ## How to use

Reviewer: ```suggestion
## How to add new icons
```

- clone this repo

```
  git clone https://github.com/kodingdotninja/chakra-icons.git
```

- go to the project directory

```
  cd chakra-icons
```

- install dependencies

```

>   yarn | yarn install | npm install

Reviewer: cause we use `yarn`. don't need `npm install`

Reviewer: ```suggestion
  yarn | yarn install
```

Author: oke I got it

```

- make a new package

```
  yarn plop
```

- fill the blank of prompts

```
  repository url : [link repo] e.g (https://github.com/twbs/icons)
  path of svg icons : [the name of folder where the svg is stored] e.g (icons)
  projects name : [name new package] e.g (bootstrap)
```

- when success generated new package, the package listed in [**./packages/@chakra-icons**](https://github.com/kodingdotninja/chakra-icons/tree/main/packages/create-chakra-icons)

- build the new package

```
  yarn build
```

> - if the process build success and no issue, then you can make a new PR

Reviewer: ```suggestion
- if the process build success and no issue, then you can make a new [PRs](https://github.com/kodingdotninja/chakra-icons/compare)
```
